#162163259 Integrate coverage service

joshtrigger opened the pull request:
**What does this PR do?**
This Pull Request integrates Coveralls code coverage service with the code base.

**Description**
The coverage service will help inform us on how much of the code is being covered by the tests.

**What are the relevant pivotal tracker stories?**
#162163259

Commit:
updated

Commit:
feat(coveralls):Integrate Coveralls code coverage service

This is to ensure that we are informed of how well the code base is being tested.

[Delivers #162163259]

Commit:
feat(coveralls):Integrate Coveralls code coverage service

This is to ensure that we are informed of how well the code base is being tested.

[Delivers #162163259]

Commit:
feat(coveralls):Integrate Coveralls code coverage service

This is to ensure that we are informed of how well the code base is being tested.

[Delivers #162163259]

joshtrigger committed:
feat(coveralls):Integrate Coveralls code coverage service

This is to ensure that we are informed of how well the code base is being tested.

[Delivers #162163259]

joshtrigger committed:
feat(coveralls):Integrate Coveralls code coverage service

This is to ensure that we are informed of how well the code base is being tested.

[Delivers #162163259]

joshtrigger committed:
feat(coveralls):Integrate Coveralls code coverage service

This is to ensure that we are informed of how well the code base is being tested.

[Delivers #162163259]

joshtrigger committed:
feat(coveralls):Integrate Coveralls code coverage service

This is to ensure that we are informed of how well the code base is being tested.

[Delivers #162163259]

joshtrigger committed:
feat(coveralls):Integrate Coveralls code coverage service

This is to ensure that we are informed of how well the code base is being tested.

[Delivers #162163259]

joshtrigger committed:
feat(coveralls):Integrate Coveralls code coverage service

This is to ensure that we are informed of how well the code base is being tested.

[Delivers #162163259]

joshtrigger committed:
feat(coveralls):Integrate Coveralls code coverage service

This is to ensure that we are informed of how well the code base is being tested.

[Delivers #162163259]

joshtrigger committed:
feat(coveralls):Integrate Coveralls code coverage service

This is to ensure that we are informed of how well the code base is being tested.

[Delivers #162163259]

joshtrigger committed:
feat(coveralls):Integrate Coveralls code coverage service

This is to ensure that we are informed of how well the code base is being tested.

[Delivers #162163259]

joshtrigger committed:
chore(readme badge): TravicCI readme badge

This readme badge will be used to display the percentage of how much the code base is being covered by the tests.

[Delivers #162163256]

joshtrigger committed:
chore(readme badge): TravicCI readme badge

This readme badge will be used to display the percentage of how much the code base is being covered by the tests.

[Delivers #162163256]

joshtrigger committed:
chore(readme badge): TravicCI readme badge

This readme badge will be used to display the percentage of how much the code base is being covered by the tests.

[Delivers #162163256]

joshtrigger committed:
chore(readme badge): TravicCI readme badge

This readme badge will be used to display the percentage of how much the code base is being covered by the tests.

[Delivers #162163256]

joshtrigger committed:
chore(coveralls):Integrate Coveralls code coverage service

This is to ensure that we are informed of how well the code base is being tested.

[Delivers #162163259]

joshtrigger committed:
chore(coveralls):Integrate Coveralls code coverage service

This is to ensure that we are informed of how well the code base is being tested.

[Delivers #162163259]

joshtrigger committed:
chore(coveralls):Integrate Coveralls code coverage service

This is to ensure that we are informed of how well the code base is being tested.

[Delivers #162163259]

joshtrigger committed:
chore(coveralls):Integrate Coveralls code coverage service

This is to ensure that we are informed of how well the code base is being tested.

[Delivers #162163259]

joshtrigger committed:
chore(coveralls):Integrate Coveralls code coverage service

This is to ensure that we are informed of how well the code base is being tested.

[Delivers #162163259]

joshtrigger committed:
chore(coveralls):Integrate Coveralls code coverage service

This is to ensure that we are informed of how well the code base is being tested.

[Delivers #162163259]

joshtrigger committed:
Merge branch 'ch-coveralls-integration-162163259' into ch-travis-readme-badge-162163256

joshtrigger committed:
chore(readme badge): TravicCI readme badge

This readme badge will be used to display the percentage of how much the code base is being covered by the tests.

[Delivers #162163256]

joshtrigger committed:
chore(readme badge): TravicCI readme badge

This readme badge will be used to display the percentage of how much the code base is being covered by the tests.

[Delivers #162163256]

joshtrigger committed:
chore(readme badge): TravicCI readme badge

This readme badge will be used to display the percentage of how much the code base is being covered by the tests.

[Delivers #162163256]

joshtrigger committed:
chore(readme badge): TravicCI readme badge

This readme badge will be used to display the percentage of how much the code base is being covered by the tests.

[Delivers #162163256]

joshtrigger committed:
chore(readme badge): TravicCI readme badge

This readme badge will be used to display the percentage of how much the code base is being covered by the tests.

[Delivers #162163256]

joshtrigger committed:
chore(readme badge): TravicCI readme badge

This readme badge will be used to display the percentage of how much the code base is being covered by the tests.

[Delivers #162163256]

joshtrigger committed:
chore(readme badge): TravicCI readme badge

This readme badge will be used to display the percentage of how much the code base is being covered by the tests.

[Delivers #162163256]

joshtrigger committed:
chore(readme badge): TravicCI readme badge

This readme badge will be used to display the percentage of how much the code base is being covered by the tests.

[Delivers #162163256]

joshtrigger committed:
Merge branch 'develop' of https://github.com/andela/ah-backend-athena into develop

joshtrigger committed:
Merge branch 'ch-travis-readme-badge-162163256' of https://github.com/andela/ah-backend-athena into ch-travis-readme-badge-162163256

joshtrigger committed:
Merge branch 'ch-coveralls-integration-162163259' of https://github.com/andela/ah-backend-athena into ch-coveralls-integration-162163259

joshtrigger committed:
chore(coveralls):Integrate Coveralls code coverage service

This is to ensure that we are informed of how well the code base is being tested.

[Delivers #162163259]

joshtrigger committed:
chore(coveralls):Integrate Coveralls code coverage service

This is to ensure that we are informed of how well the code base is being tested.

[Delivers #162163259]

KicaRonaldOkello commented:
i think it is ready to merge

joshtrigger committed:
chore(coveralls):Integrate Coveralls code coverage service

This is to ensure that we are informed of how well the code base is being tested.

[Delivers #162163259]

joshtrigger committed:
chore(coveralls):Integrate Coveralls code coverage service

This is to ensure that we are informed of how well the code base is being tested.

[Delivers #162163259]

joshtrigger committed:
chore(coveralls):Integrate Coveralls code coverage service

This is to ensure that we are informed of how well the code base is being tested.

[Delivers #162163259]

joshtrigger committed:
chore(coveralls):Integrate Coveralls code coverage service

This is to ensure that we are informed of how well the code base is being tested.

[Delivers #162163259]

collin5 commented on authors/settings.py:
```diff
@@ -78,6 +78,18 @@
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'NAME': 'athenatraviscidb',
+        'USER': 'postgres',
```
Please pick these from the environment. https://docs.travis-ci.com/user/environment-variables/#defining-variables-in-repository-settings

joshtrigger replied:
Thanks for the feedback, let me get to it immediately.

joshtrigger committed:
chore(coveralls):Integrate Coveralls code coverage service

This is to ensure that we are informed of how well the code base is being tested.

[Delivers #162163259]

joshtrigger committed:
chore(readme badge): TravicCI readme badge

This readme badge will be used to display the percentage of how much the code base is being covered by the tests.

[Delivers #162163256]

joshtrigger committed:
chore(readme badge): TravicCI readme badge

This readme badge will be used to display the percentage of how much the code base is being covered by the tests.

[Delivers #162163256]

joshtrigger committed:
chore(readme badge): TravicCI readme badge

Created environment variables on to Travis CI.

[Delivers #162163256]

collin5 reviewed (changes requested):
LGTM. Please resolve conflicts.

joshtrigger committed:
chore(coveralls):Integrate Coveralls code coverage service

resolve merge conflict

[Delivers #162163259]

joshtrigger committed:
Merge branch 'develop' of https://github.com/andela/ah-backend-athena into ch-travis-readme-badge-162163256

joshtrigger committed:
Merge branch 'ch-travis-readme-badge-162163256' into ch-coveralls-integration-162163259

joshtrigger committed:
chore(coveralls):Integrate Coveralls code coverage service

re-arrange readme badges.

[Delivers #162163259]

joshtrigger commented:
collins I implemented the feedback
